tests/test_httpclient2.rb

```diff
@@ -107,9 +111,9 @@ def test_get_pipeline
 
   def test_authheader
     EM.run {
-      setup_timeout TIMEOUT
-      EM.start_server '127.0.0.1', @port, TestServer
-      http = silent { EM::P::HttpClient2.connect '127.0.0.1', 18842 }
+      setup_timeout(windows? ? 4 : TIMEOUT)
```

Comment:
`TIMEOUT` is based on `TIMEOUT_INTERVAL`, which already a multiplier for both windows an MacOS.  Instead of supplying a straight number, what do you think about either incrementing or multiplying `TIMEOUT`?
i.e:
```suggestion
      setup_timeout(TIMEOUT + (windows? ? 2 : 0))
```
or
```suggestion
      setup_timeout(TIMEOUT * (windows? ? 1.5 : 1.0))
```

Reply:
I think I mentioned that these three tests failed on Ruby head and my Windows desktop, which is a strong system.  I just fixed the tests that failed consistently.

I think there are a lot of things in the test system that could be refactored/revised, etc.  Right now, I'm just looking to get CI passing reliably.